js/uploadbutton/Index.svelte

```diff
@@ -62,20 +63,39 @@
 	}
 </script>
 
-<UploadButton
-	{elem_id}
-	{elem_classes}
-	{visible}
-	{file_count}
-	{file_types}
-	{size}
-	{scale}
-	{min_width}
-	disabled
-	{variant}
-	{label}
-	on:click={() => gradio.dispatch("click")}
-	on:load={handle_upload}
->
-	{gradio.i18n(label)}
-</UploadButton>
+{#if mode === "static"}
+	<UploadButton
+		{elem_id}
+		{elem_classes}
+		{visible}
+		{file_count}
+		{file_types}
+		{size}
+		{scale}
+		{min_width}
+		disabled
```

Comment:
I think we can condense the code with disabled={mode === "static"}?

Reply:
Will check!

Reply:
fixd